src/formatPkg.js

```diff
@@ -402,7 +402,7 @@ function getTypes(pkg) {
   }
 
   // Check in exposed files
-  if (pkg.files && pkg.files.some(file => file.search('.d.ts'))) {
+  if (pkg.files && pkg.files.some(file => file.endsWith('.d.ts'))) {
```

Comment:
if this turns out to crash: `typeof file === "string" && file.endsWith`

Reply:
rhaa, mixed content type. Good catch